src/Core/cldj_fjx_sub_mod.F90

```diff
@@ -258,32 +265,32 @@ subroutine PHOTO_JX (U0,SZA,RFL,SOLF, LPRTJ, PPP,ZZZ,TTT,HHH,    &
            AMG(L) = (1.d0 + ZMID/RAD)**2
         enddo
       else
-           AMG(:) = 1.d0
+           AMG = 1.d0
       endif
 
 !---calculate spherical weighting functions (AMF: Air Mass Factor)
 !   indep of wavelength (refracted path assumes visible index of refraction)
 !-----------------------------------------------------------------------
       if (ATM0 .eq. 0) then
-         call SPHERE1F (U0,RAD,ZZJ,ZZHT,AMF, L1U)  ! flat Earth, AMF=1/u0
+         call SPHERE1F (U0,RAD,ZZJ,ZZHT,AMF, L1U,RC)  ! flat Earth, AMF=1/u0
```

Comment:
@lizziel: Wouldn't we need an 
```F90
IF ( RC /= CLOUDJ_SUCCESS ) THEN
  CALL CLOUDJ_ERROR( errMsg, loc, RC )
  RETURN
ENDIF
```
after each subroutine call to make sure the error gets propagated upwards?

Reply:
Argh, right! I'll add all those in.